### ISSUE TITLE
src/man/po/po4a.cfg needs to be autogenerated

### ISSUE THREAD

**Issue by pbrezina** [pbrezina/sssd-test#1214]
Cloned from Pagure issue: https://pagure.io/SSSD/sssd/issue/1210
- Created at 2012-02-27 14:23:40 by [jhrozek](https://pagure.io/user/jhrozek)
- Closed at 2020-03-24 14:21:35 as wontfix
- Assigned to nobody
---

The file `src/man/po/po4a.cfg` contains a hardcoded list of manual pages that we extract translatable strings from. This is wrong, because we forgot to update the file on multiple occasions when we added a new manual page.

The file should instead be regenerated, perhaps on every update-po or make dist.

### Comments

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2012-03-01 15:48:36](https://pagure.io/SSSD/sssd/issue/1210#221961)**

Fields changed

blockedby:  => 1219

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2012-03-01 15:49:17](https://pagure.io/SSSD/sssd/issue/1210#221962)**

Fields changed

component: SSSD => Build Environment
milestone: NEEDS_TRIAGE => SSSD 1.10 beta
owner: somebody => jhrozek
priority: major => minor

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2012-03-03 04:51:54](https://pagure.io/SSSD/sssd/issue/1210#221963)**

Fields changed

rhbz:  => 0

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2012-08-14 01:54:01](https://pagure.io/SSSD/sssd/issue/1210#221964)**

Fields changed

proposed_priority:  => Optional

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2012-09-04 23:13:35](https://pagure.io/SSSD/sssd/issue/1210#221965)**

Cleaning the 1.10 milestones before putting tickets into it.

milestone: SSSD 1.10 beta => Temp milestone

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2012-09-04 23:22:59](https://pagure.io/SSSD/sssd/issue/1210#221966)**

Fields changed

milestone: Temp milestone => SSSD 1.11 beta

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2012-09-04 23:25:14](https://pagure.io/SSSD/sssd/issue/1210#221967)**

This ticket has been evaluated for inclusion into SSSD 1.10 release and was decided to be excluded since it does not match the main goals and themes of the release. It might be considered for later releases.

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2013-01-02 15:23:36](https://pagure.io/SSSD/sssd/issue/1210#221968)**

Fields changed

milestone: SSSD 1.11 beta => SSSD 1.12 beta

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2016-02-16 14:50:37](https://pagure.io/SSSD/sssd/issue/1210#221969)**

This hasn't caused an issue since 3 years ago..

changelog:  => 
design:  => 
design_review:  => 0
fedora_test_page:  => 
mark:  => 0
milestone: SSSD 1.14 beta => SSSD Deferred
review:  => 0
selected:  => 
sensitive:  => 0

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2017-02-24 14:24:46](https://pagure.io/SSSD/sssd/issue/1210#221970)**

**Metadata Update from @jhrozek**:
- Issue assigned to jhrozek
- Issue marked as depending on: #1219
- Issue set to the milestone: SSSD Patches welcome

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2019-08-28 15:40:51](https://pagure.io/SSSD/sssd/issue/1210#593230)**

**Metadata Update from @jhrozek**:
- Assignee reset

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2020-03-24 14:21:34](https://pagure.io/SSSD/sssd/issue/1210#636984)**

Thank you for taking time to submit this request for SSSD. Unfortunately this issue was not given priority and the team lacks the capacity to work on it at this time.

Given that we are unable to fulfill this request I am closing the issue as wontfix.

If the issue still persist on recent SSSD you can request re-consideration of this decision by reopening this issue. Please provide additional technical details about its importance to you.

Thank you for understanding.

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2020-03-24 14:21:36](https://pagure.io/SSSD/sssd/issue/1210#636985)**

**Metadata Update from @pbrezina**:
- Issue close_status updated to: wontfix
- Issue status updated to: Closed (was: Open)
